src/quo2/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.avatars.wallet-user-avatar.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.wallet-user-avatar.style :as style]
+            utils.string))
+
+(def circle-sizes
+  {:small   20
+   :medium  32
+   :large   48
+   :x-large 80})
+
+(def font-sizes
+  {:small   :label
+   :medium  :paragraph-2
+   :large   :paragraph-1
+   :x-large :heading-1})
+
+(def font-weights
+  {:small   :medium
+   :medium  :semi-bold
+   :large   :semi-bold
+   :x-large :medium})
+
+(defn- view-internal
+  "Options:
+
+  :full-name - string (default: nil) - used to generate initials
+  :color  - keyword (default: nil) - color of the avatar
+  :size   - keyword (default: :x-large) - size of the avatar"
+  [{:keys [full-name color size theme]
+    :or   {full-name "empty name"
+           color     :red
+           size      :x-large}}]
```

Comment:
We recently update the approach to sizes. Could you check my open pr that has the updated docs on this?
Basically take the number from figma and make it a key with size at the beginning. ':size-<number>' e.g :size-20

Reply:
Please check the end result to make sure it's ok:

```
(def properties
  {:size-20 {:size        20
             :font-size   :label
             :font-weight :medium}
   :size-32 {:size        32
             :font-size   :paragraph-2
             :font-weight :semi-bold}
   :size-48 {:size        48
             :font-size   :paragraph-1
             :font-weight :semi-bold}
   :size-64 {:size        64
             :font-size   :paragraph-1
             :font-weight :medium}
   :size-80 {:size        80
             :font-size   :heading-1
             :font-weight :medium}})

```

Reply:
yep looks good 👍